### PR TITLE
Announce Virtual Comp, Mentoring, and Rules

### PR DESCRIPTION
**Pull request by Adimote**
Add an email announcing the virtual competition

TODO:
- [x] Add a livestream link (once we have a livestream set up)
- [x] Add a link to the event page on the site once we have that up. 
- [x] Add the rules revision to the bottom

### CODE REVIEW

File: SR2024/2023-11-08-virtual-competition-announcement.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+Hello! 
+
+We're happy to announce that in addition to our physical competition, we'll be holding a **virtual competition**. We'll run our virtual competition on our [robot simulator](https://studentrobotics.org/docs/simulator/).
+
+The Virtual competition is an opportunity for your teams to test their coding skills before the physical competition takes place.
+
+The results will be broadcast live on **Saturday 9th March 2024 at 1pm** via a livestream on our YouTube channel.
+
+The code submission deadline will be **2 days before**, that's Thursday 7th March 2024 at 6pm.
```
Comment (PeterJCLaw) [inline SR2024/2023-11-08-virtual-competition-announcement.md:14]: Clarify that this is UK time?

**Review by Adimote — COMMENTED** [review]
I've pulled the changes from #167 into this

File: SR2024/2023-11-08-virtual-competition-and-mentoring.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+Please fill out the form below to register an interest in mentoring. While we can't guarantee regular support, we'll do our best to match you with a volunteer as soon as we're able.
+
+https://forms.gle/V831FNHaMLC4h4pVA
+
+We would also like one of your students to post a short report in your team's Discord channel after they meet so we are better able to offer help. If you/your teams are not yet in [Discord](https://studentrobotics.org/docs/tutorials/discord.html) and you need help, please let us know.
+
+# Rules amendments
+
+We've also made some minor amendments to the rules which are worth noting:
```
Comment (PeterJCLaw) [inline SR2024/2023-11-08-virtual-competition-and-mentoring.md:38]:
```suggestion
We've also made some minor amendments to the [rules](https://srobo.org/rules/) which are worth noting:
```

File: SR2024/2023-11-08-virtual-competition-and-mentoring.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
+---
+to: Student Robotics 2024
+subject: SR2024 Virtual Competition, Mentoring, and rules changes
```
Comment (PeterJCLaw) [inline SR2024/2023-11-08-virtual-competition-and-mentoring.md:3]:
```suggestion
subject: SR2024 Virtual Competition, Mentoring, and Rules changes
```

File: SR2024/2023-11-08-virtual-competition-and-mentoring.md [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+---
+
+Hello! 
+
+We hope your team found Kickstart useful and are excited for the year ahead - we certainly are, and we have a couple things to announce that will happen along the way:
```
Comment (PeterJCLaw) [inline SR2024/2023-11-08-virtual-competition-and-mentoring.md:8]:
```suggestion
We hope your team found Kickstart useful and are excited for the year ahead - we certainly are, and we have a few things to announce that will happen along the way:
```

File: SR2024/2023-11-08-virtual-competition-and-mentoring.md [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+
+# Virtual Competition
+
+We're happy to announce that in addition to our physical competition, we'll be holding a **virtual competition**. We'll run our virtual competition on our [robot simulator](https://studentrobotics.org/docs/simulator/).
+
+The virtual competition is an opportunity for your teams to test their coding skills before the physical competition takes place.
+
+The matches will be broadcast live on **Saturday 9th March 2024 at 1pm GMT**  via a [livestream](https://www.youtube.com/watch?v=hlfaQIfLaRg) on our YouTube channel.
+
+The code submission deadline will be **2 days before**, that's Thursday 7th March 2024 at 6pm GMT. Your team can submit their robot code to [the code submitter](https://studentrobotics.org/code-submitter/). The log in details have the username of your team's three letter acronym (in upper case): <TLA mailmerges in here>, and the password is the same as the password your team used to enter Discord.
```
Comment (PeterJCLaw) [inline SR2024/2023-11-08-virtual-competition-and-mentoring.md:18]:
```suggestion
The code submission deadline will be **two days before**, that's Thursday 7th March 2024 at 6pm GMT. Your team can submit their robot code to [the code submitter](https://studentrobotics.org/code-submitter/). The log in details have the username of your team's three letter acronym (in upper case): <TLA mailmerges in here>, and the password is the same as the password your team used to enter Discord.
```

File: SR2024/2023-11-08-virtual-competition-and-mentoring.md [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+
+During the competition, your robot will compete in (at least) 4 virtual matches for real league points, these points will then transfer across to the physical competition!
```
Comment (prophile) [inline SR2024/2023-11-08-virtual-competition-and-mentoring.md:20]:
```suggestion
During the competition, your robot will compete in a number of virtual matches for real league points, these points will then transfer across to the physical competition!
```

File: SR2024/2023-11-08-virtual-competition-and-mentoring.md [hunk 7]
```diff
@@ -0,0 +1,37 @@
...
+
+Hello! 
+
+We hope your team found Kickstart useful and are excited for the year ahead - we certainly are, and we have a couple things to announce that will happen along the way:
+
+# Virtual Competition
+
+We're happy to announce that in addition to our physical competition, we'll be holding a **virtual competition**. We'll run our virtual competition on our [robot simulator](https://studentrobotics.org/docs/simulator/).
+
+The virtual competition is an opportunity for your teams to test their coding skills before the physical competition takes place.
```
[the remark this answers is not in the document]
Comment (PeterJCLaw) [reply SR2024/2023-11-08-virtual-competition-and-mentoring.md]: https://github.com/srobo/team-emails/pull/164/commits/cfed3c7ad7ef5f0879dc4865a71fb0b85d6d2605, perhaps?

**Review by prophile — DISMISSED** [review]

**Review by prophile — DISMISSED** [review]

**Pull request merged**